### DISCUSSION TITLE
Converting the First Course into an Audiobook

### DISCUSSION THREAD

**Discussion by tanmaylaud** [OpenMined/courses#220] [Feedback] [answered]
The first course is mainly audio explanation since there are no visualisations being displayed. 
This provides a really good opportunity to convert this course into an Audiobook which will help us reach more students across the globe. 

Imagine learning about PrivacyAI while going out for a walk ;) . 
It will also help those who are visually challenge or those who do not have time to be on the laptop all the time. 

Ofcourse, the quizzes and final assignment will be missed but we can figure out something :) .

@iamtrask

**Comment by hershd23** [comment]
I second that!

**Comment by imflash217** [comment]
Agreed. Mostly because Course-1 does not have much visual technical element in it.

**Comment by yemikifouly** [comment]
I am adding this as a [feature request](https://github.com/OpenMined/courses/issues/243) so the rest of the team can consider it. I believe there are already a few things already in the work for the course including: 

* Writing out the transcript of the whole course.
* Translating the transcript to several languages.

So, I will add this as it is a great idea! 🙂 Thanks for suggesting Tanmay.

**Reply by tanmaylaud** [reply]
Thanks @yemikifouly ! Let me know if any help needed

**Comment by tanmaylaud** [comment, marked as answer]
shall continue the discussion [here](https://github.com/OpenMined/courses/issues/243)
